app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -46,8 +46,12 @@ def create
         locale: params[:marketplace_language]},
         marketplace[:id]).data
 
+    # make the marketplace creator land to the admin panel
+    url = "#{marketplace[:url]}/admin"
```

Comment:
I was not very confident when choosing the right place where the /admin should be added (to make new admin land to Onboarding Wizard instead of the Confirmation Pending page).
I chose the controller level, while other possible options would have been in the marketplace service (but if felt better that it returns the normal base url) and modifying the URL at marketing site, but I thought it's better that marketing site doesn't need to know about URL changes in R&A.
So controller feels best, but I hope the reviewer to give second opinion on this choice.